en/docs/learn/design-api/lifecycle-management/customize-api-life-cycle.md

```diff
@@ -143,72 +130,76 @@ Follow the steps below to add a new state to the default life cycle.
 
-         ``` java
-         ...   
-            <transition event="Reject" target="Rejected"/>
-         ...
-         ```
-
-8.  Make the transition event visible in API Publisher.
-     
-     1.  Open the `<API-M_HOME>/repository/deployment/server/jaggeryapps/publisher/site/conf/locales/jaggery/locale_default.json` file.
-    
-     2.  Add the following code in the file. Note that the key value in the JSON pair should be lowercase.
-
         ``` java
-        "reject": "Rejected",
+        ...   
+        <transition event="Reject" target="Rejected"/>
+        ...
         ```
 
-9.  Restart the WSO2 API Manager server.
+7.  Restart the WSO2 API Manager server.
+
+8.  Open API Publisher and check the Lifecycle tab to see the newly added state.
+
+    ![]({{base_path}}/assets/img/learn/custom-life-cycle-states.png)
+
+    !!! tip
+        Consider the following points when extending and customizing the API lifecycle XML configuration.
+
+        -   Do not change the life cycle name since it needs to be engaged with the APIs dynamically.
+        -   Make sure you keep the **PUBLISHED** and **PROTOTYPED** states as those two states will be used by API Publisher in the API creation wizard.
+
+    !!! info
+        By default following lifecycle diagram is added in Publisher portal to show the state changes.
+
+        ![]({{base_path}}/assets/img/learn/lifecycle-image.png)
 
-10.  Re-open API Publisher and check the Lifecycle to see the changes.
+        If you want to change the lifecycle image in Publisher, you can follow the steps mentioned below:
 
-     ![]({{base_path}}/assets/attachments/103328609/103328611.png)
+        -   Search for **lifeCycleImage** in `defaultTheme.js` file resides in `<APIM-Home>/repository/deployment/server/jaggeryapps/publisher/site/public/conf` directory and uncomment it.
+        -   Replace the path with correct path of image. For instance,
 
-!!! tip
-    Consider the following points when extending and customizing the API lifecycle XML configuration.
+                lifeCycleImage: '/publisher/site/public/images/custom-lifecycle.png,
 
--   Do not change the life cycle name since it needs to be engaged with the APIs dynamically.
--   Make sure you keep the **PUBLISHED** and **PROTOTYPED** states as those two states will be used by API Publisher in the API creation wizard.
+    !!! Attention "For Advanced Customization"
+        If you want to add the custom lifecycle state to the existing lifecycle diagram, you have to customize the [LifeCycleImage.jsx](https://github.com/wso2/carbon-apimgt/blob/v6.6.163/features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/LifeCycle/LifeCycleImage.jsx) file as mentioned in the [Advanced Customization]({{base_path}}/learn/consume-api/customizations/customizing-the-developer-portal/advanced-customization/) section.
```

Comment:
Since we are shipping the source from 3.1.0, we can point to the JSX file of the publisher instead of the git url.

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/951/commits/1eb75c94dacd171c72a817499d0692eea68f74bc